src/components/20-molecules/dropdown/ui.test.js

```diff
@@ -6,48 +6,41 @@ fixture('Dropdown').page(`${host}/iframe.html?id=molecules-dropdown--dropdown`);
 
 test('should render correctly', async t => {
   const dropdown = await Selector(() =>
-    document.querySelector(`axa-dropdown[data-test-id="dropdown"]`)
+    document.querySelector(`axa-dropdown`)
   );
   await t.expect(dropdown.exists).ok();
 });
 
-fixture('Dropdown valid').page(
-  `${host}/iframe.html?id=molecules-dropdown--dropdown-w-valid-checkmark`
+fixture('Dropdown check mark').page(
+  `${host}/iframe.html?id=molecules-dropdown--dropdown&knob-checkmark=true`
 );
 test('should show checkmark', async t => {
   const dropdownCheckmark = await Selector(() =>
     document.querySelector(
-      `axa-dropdown[data-test-id="dropdown-valid"] .m-dropdown__valid-icon .m-dropdown__valid-icon-inner-active`
+      `axa-dropdown .m-dropdown__check`
     )
   );
   await t.expect(dropdownCheckmark.exists).ok();
 });
 
 fixture('Dropdown error').page(
-  `${host}/iframe.html?id=molecules-dropdown--dropdown-w-error-message`
+  `${host}/iframe.html?id=molecules-dropdown--dropdown&knob-invalid=true&knob-error=error`
 );
 
-test('should show error message and correct colors', async t => {
-  const dropdown = await Selector(() =>
-    document.querySelector(`axa-dropdown[data-test-id="dropdown-error"]`)
+test('should show error message and have the correct color', async t => {
+  const $axaError = await Selector(() =>
+    document.querySelector(`axa-dropdown .m-dropdown__error `)
```

Comment:
I would recommend using `js-dropdown__error` instead of `m-dropdown__error`.

Reply:
good catch! Done.